DNN Platform/Tests/DotNetNuke.Tests.Core/Controllers/Search/InternalSearchControllerTests.cs

```diff
@@ -250,7 +250,7 @@ private IDataReader GetPortalCallBack(int portalId, string culture)
             table.Rows.Add(portalId, null, "My Website", "Logo.png", "Copyright 2011 by DotNetNuke Corporation", null,
                 "2", "0", "2", "USD", "0", "0", "0", "0", "0", "1", "My Website", "DotNetNuke, DNN, Content, Management, CMS", null,
                 "1057AC7A-3C08-4849-A3A6-3D2AB4662020", null, null, null, "0", "admin@change.me", "en-US", "-8", "58", "Portals/0",
-                null, homePage.ToString("D"), null, null, "57", "56", "-1", "-1", "7", "-1", "2011-08-25 07:34:11", "-1", "2011-08-25 07:34:29", culture);
+                null, homePage.ToString("D"), null, null, "57", "56", "-1", "-1", "7", null, null, "-1", "2011-08-25 07:34:11", "-1", "2011-08-25 07:34:29", culture);
```

Comment:
Is the expected value actually null here, or -1 like the other values?

Reply:
Disclaimer: I have not been able to get testing running on my VS Community 2017 locally. So I'm reaching in the dark. In any case: both -1 and null should return the same behavior in the site. These are nullable fields (during an upgrade existing portals will have null on these fields) so I figured it'd be the same.